docs/releases/1.19-NOTES.md

```diff
@@ -4,6 +4,8 @@
 
 # Significant changes
 
+* **The default image has been updated to Ubuntu 20.04 (Focal)**. Consequently, the SSH user changed to `ubuntu` and the Linux kernel changed to version 5.4.
```

Comment:
Is this a breaking change?

Reply:
It will be the first one listed in the 1.18 release notes at significant. Depending on how you use Kops, this could break things, but not sure how likely considering Debian and Ubuntu are quite similar.